lms/djangoapps/learner_recommendations/serializers.py

```diff
@@ -0,0 +1,43 @@
+"""
+Serializers for course about page recommendations.
+"""
+from rest_framework import serializers
+
+
+class ActiveCourseRunSerializer(serializers.Serializer):
+    """Serializer for active course run for course about page recommendations API"""
+    key = serializers.CharField()
+    type = serializers.CharField()
+    marketingUrl = serializers.URLField(source="marketing_url")
+
+
+class CourseOwnersSerializer(serializers.Serializer):
+    """Serializer for course owners for course about page recommendations API"""
+    key = serializers.CharField()
+    name = serializers.CharField()
+    logoImageUrl = serializers.URLField(source="logo_image_url")
+
+
+class RecommendedCourseSerializer(serializers.Serializer):
+    """Serializer for a recommended course from the recommendation engine"""
+
+    uuid = serializers.CharField()
+    title = serializers.CharField()
+    image = serializers.CharField()
```

Comment:
image is a dict with `src` as a key.
```
image: {
  src: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
},
```

Reply:
This is the data we are getting in API right now:
```python
"courses":[
    {
        "uuid":"d61d7a1f-3333-4169-a786-92e2bf690c6f",
        "title":"IELTS Academic Test Preparation",
        "image":{
            "src":"https://prod-discovery.edx-cdn.org/media/course/image/d61d7a1f-3333-4169-a786-92e2bf690c6f-fa8a6909baec.small.jpg",
            "description":null,
            "height":null,
            "width":null
        },
        "owners":[
            {
                "key":"UQx",
                "name":"The University of Queensland",
                "logo_image_url":"https://prod-discovery.edx-cdn.org/organization/logos/8554749f-b920-4d7f-8986-af6bb95290aa-f336c6a2ca11.png"
            }
        ],
        "prospectus_path":"courses/ielts-academic-test-preparation",
        "active_course_run":{
        "key":"course-v1:UQx+IELTSx+1T2022",
        "type":"Active",
        "marketing_url":"https://www.edx.org/course/ielts-academic-test-preparation-course-v1uqxieltsx1t2022?utm_source=lms_catalog_service_user&utm_medium=affiliate_partner"
    }
},
```

@syedsajjadkazmii do you need other fields of image object: `description`, `height` and `width` on FE?

Reply:
@zainab-amir no i just need the src